src/error/errorRender.ts

```diff
@@ -0,0 +1,14 @@
+import { ErrorOut, errors, KoaContextWithOIDC } from "oidc-provider";
+import { logger } from "../logger.js";
+
+export const errorRender = (
+  ctx: KoaContextWithOIDC,
+  out: ErrorOut,
+  error: errors.OIDCProviderError | Error
+): void => {
+  if (error instanceof errors.OIDCProviderError) {
+    logger.error(
+      `OIDC-ProviderError: ${error.error}: Description '${error.error_description}',  Detail: '${error.error_detail ?? ""}'`
+    );
+  } else logger.error(`Unknown Error occurred: ${error.message}`);
+};
```

Comment:
Is this still going to return the response?

Reply:
This does still return the response but this just overwrites the default behaviour which serves html to the user on an error:  https://github.com/panva/node-oidc-provider/tree/main/docs#rendererror.